### ISSUE TITLE
taxon constraint and request for definition clarification

### ISSUE THREAD

**Issue by michaelerice** [geneontology/go-ontology#12590]
request following terms to be restricted to:  never_in   taxon:8782 Aves
1) GO:0044305 calyx of Held
 avians do not have a homolog for the MNTB, the nucleus where the calyx of Held is located. The organization of synaptic inhibition is different in avians, cf. Fig. 2 PMID: 21525367
2) GO:0070342    brown fat cell proliferation
 GO:0050873  brown fat cell differentiation
cf. PMID: 21695679

request GO:0061010 gall bladder development be restricted to never_in taxon:  8801 Struthio camelus
and never_in taxon: 8930 Columbidae
Pigeons, most parrots and ostriches do not have gall bladders.
Avian Medicine and Surgery in Practice
Written By: Doneley, Bob
Published By: CRC Press
Published In: 2016

Request review of definition for:  

1)"capacitation" for term GO:0048240    sperm capacitation
"avian spermatozoa do not require a period of capacitation, since they are competent to fertilize
the egg without a prolonged sojourn in the oviduct or in a 'capacitation medium' in vitro."
Fertilization in Protozoa and Metazoan Animals: Cellular and Molecular Aspects
by Tarín, Juan J; Cano, Antonio 2000
if capacitation is defined as a process that happens in the avian female reproductive tract, then this term should be restricted to never_in     taxon:8782 Aves. But if capacitation can include processes that occur prior to being deposited from the avian male then the term could possibly be applied to this taxon.

2) GO:0043084 penile erection
Basal birds (paleognaths) and waterfowl (anseriforms)
retain a well-developed phallus, although in most birds
the phallus is significantly reduced or absent [King, 1981;
Briskie and Montgomerie, 1997].
 penile erection ......birds use lymphatics
PMID: 25115961 

Even birds wit a penis would not fit  this term because the definition specifies "Achieved by increased inflow of blood into the vessels of erectile tissue"

**Comment by cmungall** [comment]
> GO:0044305 calyx of Held

agreed. Do you thing we should also have a link to [UBERON:0002833](http://purl.obolibrary.org/obo/UBERON_0002833) ! medial nucleus of trapezoid body 

(from where we should inherit the TC)

> gall bladder, never in pigeons or ostriches

Added to uberon, thanks, will be inherited

> penile erection in aves

well spotted! We need the loosen this up so to speak. The part about erectile tissue is still correct?

note to self, "The erection mechanism of the ratite penis", add some of these classes to uberon:
http://prumlab.yale.edu/sites/default/files/brennan_prum_2011.pdf

**Comment by paolaroncaglia** [comment]
Added taxon constraints: 
GO:0044305 calyx of Held never_in taxon:8782 Aves [dbxref PMID: 21525367]
GO:0070342 brown fat cell proliferation never_in taxon:8782 Aves [dbxref PMID: 21695679]
GO:0050873 brown fat cell differentiation never_in taxon:8782 Aves [dbxref PMID: 21695679]

Added definition comment to GO:0048240 sperm capacitation 
[Avian spermatozoa do not require a period of capacitation, since they are competent to fertilize the egg without a prolonged sojourn in the oviduct or in a 'capacitation medium' in vitro. Therefore, ’sperm capacitation’ can only be applied to Aves to refer to gene products involved in processes that occur prior to sperm being deposited from the avian male. The term cannot apply to processes occurring in the avian female reproductive tract.]
[dbxref ISBN 978-3-642-58301-8]

Assigning to @cmungall for any remaining edits (wrt a) link to UBERON:0002833 ! medial nucleus of trapezoid body, b) loosening up def of GO:0043084 penile erection and c) related addition of UBERON classes. Thanks.

**Comment by michaelerice** [comment]
---------- Forwarded message ----------
From: Froman, David <david.froman@oregonstate.edu>
Date: Tue, Aug 23, 2016 at 3:29 PM
Subject: RE: question: sperm capacitation in birds
To: Fiona McCarthy <fionamcc@email.arizona.edu>


Your definition of "capacitation" is about the same as it was when I was a post-doc at CSU working with mammalian sperm.  It's there, it's real, but nobody has defined it in the sense that sickle cell anemia stems from substituting one amino acid for another in hemoglobin, or so I understand.  So there seem to be a number of things that happen, perhaps a shift in the phospholipid to cholesterol ratio and other stuff that cumulatively enable "capacitation."

Birkett Howarth, who was a professor of Poultry Science at UGA, published an experiment in the early 1980s, I think in Biology of Reproduction . . . maybe 1983 . . . where he demonstrated that capacitation, as explained for mammals, was highly unlikely.

Howarth also did experiments in which testicular sperm are inseminated into the magnum via laparotomy.  I have done these too.  You get fertilized eggs out the wahzoo. One could argue that capacitation occurs while sperm where in the infundibulum.  That said, it is unlikely that the surface of the infundibulum approximates the vaginal surface.

One could also argue that sperm sequestration within the sperm storage tubules illustrates an entirely different, perhaps more ancient, reproductive strategy.  When I was a grad student, I always thought is was intellectually lame to attempt to explain poultry reproduction in terms of what was known about mammals.

In summary, in my professional opinion, it is highly unlikely that chicken sperm undergo capacitation.  As for "all avian species," I doubt if the question has ever been resolutely studied.  Rather, them damn chikkens serve as exemplar of all things feathered.

**Comment by michaelerice** [comment]
Considering the information provided by David Froman and the statement found on p.241 of [dbxref ISBN 978-3-642-58301-8] Fiona McCarthy says the term GO:0048240 sperm capacitation  should be restricted to never_in taxon: 9397 bats, never_in taxon: 1294634 Crocodylia, never_in taxon: 8504 
Lepidosauria, never_in taxon: 8459 Testudines, never_in taxon:8782 Aves

**Comment by dosumis** [comment]
All taxon restrictions on anatomical structures on anatomical structures are now inherited from Uberon.  This ticket therefore belongs on the [Uberon tracker](https://github.com/obophenotype/uberon/issues).

**Comment by michaelerice** [comment]
?David,


Do I need to open a ticket on Uberon tracker or will it be transferred automatically? Thank you


MER


________________________________
Michael E. Rice AgBase biocurator
School of Animal and Comparative Biomedical Sciences
Thomas W. Keating Bioresearch Building
1657 E. Helen St.
Tucson, AZ 85721
The College of Agriculture and Life Sciences
The University of Arizona
520 955-2745
mrice@email.arizona.edu<http://mrice@email.arizona.edu>
________________________________

**Comment by cmungall** [comment]
Usually I will open a separate ticket. In this case I just went ahead and added the two  TCs for glallbladder. If you scroll up to where it says "added a commit to obophenotype/uberon that referenced this issue 27 days ago" you can see the change. Sorry, I will be clearer next time.

The calyx of held/medial nucleus of trapezoid body is trickier, not clear what the best place is to make TCs

**Comment by pgaudet** [comment]
@michaelerice Are there still tasks to be done for this ticket ? 

Thanks, Pascale
